### DISCUSSION TITLE
Miner Setup (Lotus.filecoin.io - Tutorials)

### DISCUSSION THREAD

**Discussion by Sangmin1514** [filecoin-project/lotus#9566] [Tutorials]
https://lotus.filecoin.io/tutorials/lotus-miner/run-a-miner/

Initialize the miner : 
lotus-miner init --owner=<address>  --worker=<address> --no-local-storage

After entering this part, the following window appears.

Worker: 0; Base: 0; Target: 2281904 (diff: 2281904)
State: header sync; Current Epoch: xxxxxxx; Todo: xxxxxx
Validated 0 messages (0 per second)

Current Epoch and Todo are constantly changing numbers.
It's been 3 days with this screen.
Is it a long-running job?

**Comment by rjan90** [comment]
Hey @Sangmin1514! It seems like you are trying to sync the full chain. I recommend you try to sync from a snapshot: https://lotus.filecoin.io/lotus/manage/chain-management/#lightweight-snapshot

**Reply by Sangmin1514** [reply]
Thanks you! :)
